AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestRunnerService.kt

```diff
@@ -47,6 +48,20 @@ interface TestRunnerService {
         bytes: suspend () -> ByteArray
     ): UploadedApk
 
+    /**
+     * Finds the APK in Google Cloud Storage.
+     * If it doesn't exist, copy the data from the source GCS path to the target path.
+     *
+     * @param sourceGcsPath the GCS path that we may need to copy file from
+     * @param targetRelativePath the GCS that we would write the file to
+     *
+     * @return Returns an [RemoteApk] instance that identifies the file in the cloud.
+     */
+    suspend fun getOrUploadRemoteApk(
```

Comment:
lets document the behavior here.
as in, i'm guessing this will not copy if the targetRelativePath already exists. (e.g. we won't check the file hasn't changed)

Reply:
done